src/flutter/shell/platform/linux_embedded/surface/linuxes_surface_gl_drm.h

```diff
@@ -5,62 +5,105 @@
 #ifndef FLUTTER_SHELL_PLATFORM_LINUX_EMBEDDED_SURFACE_SURFACE_GL_DRM_H_
 #define FLUTTER_SHELL_PLATFORM_LINUX_EMBEDDED_SURFACE_SURFACE_GL_DRM_H_
 
-#include <gbm.h>
-
 #include <memory>
 
-#include "flutter/shell/platform/linux_embedded/surface/context_egl_drm.h"
+#include "flutter/shell/platform/linux_embedded/logger.h"
 #include "flutter/shell/platform/linux_embedded/surface/linuxes_egl_surface.h"
 #include "flutter/shell/platform/linux_embedded/surface/linuxes_surface.h"
 #include "flutter/shell/platform/linux_embedded/surface/linuxes_surface_gl_delegate.h"
+#include "flutter/shell/platform/linux_embedded/window/native_window_drm.h"
 
 namespace flutter {
 
-class SurfaceGlDrm final : public Surface<gbm_surface>,
-                           public SurfaceGlDelegate {
+template <typename C>
```

Comment:
`template <typename C>` -> `template <typename T>`

Reply:
I fixed it.